runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg

```diff
@@ -1,21 +1,17 @@
 TestType() ::= "Lexer"
 
-Options ::= [
-	"Ignore": "Currently unable to generate the required CRLF input sequence."
-]
-
 Grammar ::= [
 	"L": {<grammar("L")>}
 ]
 
-Input() ::= "34<\u000D><\n> 34"
```

Comment:
It’s been like that for the past 6 months, after discussing with Ter.
Right now we don’t have \r tests since these tests are ignored
So better loose 50% coverage than 100%

> Le 21 juin 2015 à 21:51, Sam Harwell <notifications@github.com> a écrit :
> 
> In runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg <https://github.com/antlr/antlr4/pull/927#discussion_r32893653>:
> 
> >  Grammar ::= [
> >  	"L": {<grammar("L")>}
> >  ]
> >  
> > -Input() ::= "34<\u000D><\n> 34"
> @ericvergnaud <https://github.com/ericvergnaud> I am not opposed to separating the tests for \r from the tests for other items. However, the tests were not originally created with this separation; this proposed change removes the \r tests which we did have and does not replace them with an alternative.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/927/files#r32893653>.
>